jedi/evaluate/docstrings.py

```diff
@@ -29,11 +29,13 @@
     r'\s*:type\s+%s:\s*([^\n]+)',  # Sphinx
     r'\s*:param\s+(\w+)\s+%s:[^\n]+',  # Sphinx param with type
     r'\s*@type\s+%s:\s*([^\n]+)',  # Epydoc
+    '\s*%s\s*\((.+?)\):.+',  # Google param with type
```

Comment:
This doesn't check for `Args`. Shouldn't we do that? Otherwise it might just much 

Also I don't think you need `.+`.